### DISCUSSION TITLE
I want to install phidgets vcp1001 on my artisan and draw a graph of the voltages.

### DISCUSSION THREAD

**Discussion by hato-sable99** [artisan-roaster-scope/artisan#1523] [Q&A]
I want to install phidgets vcp1001 on my artisan and draw a graph of the voltages.
I attached the HUB0002 and VCP1001 to the PC and recognized them with the extra device. 
Voltage numbers appear on the right side of the graph area, but the voltage graph is not drawn. 
My roasting machine is a BESCA BSC-03 manual.　
The exhaust fan and fire control of the BSC-03 are controlled by a voltage of 0V-10V.
I want to draw a graph of the exhaust vs. the heat level.Please help me.
![IMG20240326143711](https://github.com/artisan-roaster-scope/artisan/assets/161304768/16458576-d97c-40ae-8557-1a66760eab46)

**Comment by hato-sable99** [comment]
Please let me know what formula to enter in the additional device field.

**Reply by MAKOMO** [reply]
Your "Exhaust Volt" values are most likely out of the range of your temperature axis limits (not visible in your picture, but maybe max out at 300C)!?

Depending on what the expected number range for the value of this channel are, you could put a simple x/10. Depending on how those values change over the roast, you might also define a [quantifier](https://artisan-roasterscope.blogspot.com/2014/04/event-quantifiers.html) to map those values to discrete events.

**Reply by hato-sable99** [reply]
Thanks for the solution.
I entered x/100 and got an accurate graph.
The graph is drawn between 0°C and 100°C as the voltage changes from 0v to 10v.　
This is a very nice feature.　
Appreciate your response.
